ckanext/scheming/validation.py

```diff
@@ -557,3 +556,42 @@ def repeating_text_output(value):
         return json.loads(value)
     except ValueError:
         return [value]
+
+
+@scheming_validator
```

Comment:
this is only for validators that are passed the schema information

Reply:
Thanks